angr/analyses/reaching_definitions/engine_vex.py

```diff
@@ -492,9 +491,40 @@ def _handle_CCall(self, expr):
     #
 
     def _handle_function(self, func_addr: Optional[DataSet], **kwargs):
+        if func_addr is not None and len(func_addr) == 1 and self.functions is not None:
+            func_addr_int: Union[int,Undefined] = next(iter(func_addr))
+            if self.functions.contains_addr(func_addr_int):
+                _cc = self.functions[func_addr_int].calling_convention or cc
+        cc: SimCC = _cc or DEFAULT_CC.get(self.arch.name, None)(self.arch)
```

Comment:
I don't think this will work -- the first access to `_cc` will lead to an access to an undefined variable. You should try to run this function first :)